katz-rx/src/main/kotlin/katz/rx/observable/instances/ObservableWrap.kt

```diff
@@ -11,13 +11,9 @@ data class ObservableWrap<A>(val value: Observable<A>) : ObservableWrapKind<A> {
 
     class F private constructor()
 
-    inline fun <B> map(crossinline f: (A) -> B): ObservableWrap<B> =
-            ObservableWrap(value.map({ f(it) }))
+    fun <B> map(f: (A) -> B): ObservableWrap<B> =
+            ObservableWrap(value.map(f::invoke))
```

Comment:
is there a point to have `inline fun ...( noinline f)`? here `f` is the only lambda parameter so both keywords would cancel each other, am I right?